subsys/bluetooth/host/ecc.h

```diff
@@ -23,9 +23,8 @@ struct bt_pub_key_cb {
 
 /*  @brief Generate a new Public Key.
  *
- *  Generate a new ECC Public Key. The callback will persist even after the
- *  key has been generated, and will be used to notify of new generation
- *  processes (NULL as key).
+ *  Generate a new ECC Public Key. The callback will persist until the key
+ *  has been generated. Subsequent calls will overwrite the stored callback.
```

Comment:
I'd change this to something like:
Generate a new ECC Public Key. Provided cb must persists until callback is called. Function with same cb parameter shall not be called again until callback is called.